plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt

```diff
@@ -132,19 +132,20 @@ class BuildProgressSplitterPanelManager(private val project: Project) :
         }
 
         if (newState in setOf(
+                TransformationStatus.ACCEPTED,
```

Comment:
In the right panel, we show a message saying transformation "has been accepted," so we should add this here to match the left and right sides of the Transformation Hub